### DISCUSSION TITLE
Feature request: Littermates changes

### DISCUSSION THREAD

**Discussion by Pletoo** [sheltermanager/asm3#1509] [Ideas]
The "littermates" summary screen is so useful as a quick way to check up on a litter and navigate to their records. It would be nice if the mother's record was also included on this screen (without making her a littermate.) Preferably the mother would be at the at the top with a small spacer before the kitten records. Also helpful to have the mother included on the shelter view sort by litter. This can sort of be done through the edit litter screen but that doesn't include the emblems on the mother's name to show her status like it does on the kitten names.

I'd also like an option to select whether it was a litter, an intake group, or colony. Our shelter only deals with TNR, stray, or abandoned cats in a rural farming community, so that means we often get groups of cats that are not necessarily a litter or even related to each other, but may have similar medical issues that we need to track, It would be a lot faster if I could pull up a group summary screen just like the littermates screen (and less confusing if I could call it a colony or group instead of a litter.)

Slightly separate issue...can we have an CSV import field for litter number, please? It is so nice to do one import for 40+ colony cats but I'd like to also be able to identify litters with that same import.

Thank you

**Comment by bobintetley** [comment]
Hi, the mother is also shown in the banner at the top of the screen when editing an animal that belongs to a litter. I'll make a note that emblems aren't appearing for the mother in the edit litter screen and add a ticket for csv import of the litter reference.

As long as you don't create an actual litter record in the edit litters screen (as it's this that is used to calculate litter figures for reporting), there's nothing to stop you using the litter reference field on animal records to group a bunch of them together with the littermates button. I don't think duplicating the same functionality and calling it group or colony is going to offer anything in terms of recording or reporting.

Thanks again for giving us feedback and ideas!

**Reply by Pletoo** [reply]
Hi,
I was mainly looking for a way to get directly from the mother's record to the kitten records instead of going through the edit litter screen. When we get a pregnant or nursing mother in, she doesn't get vaccinated/ altered until the kittens weaned (or deceased) so it is helpful in scheduling a vet visit when their records are more visibly connected on one screen. I will continue using the edit litter screen and having the mother's emblems on that screen will definitely help.

I was imagining just having a dropdown box on the add litter screen (editable as lookup data?) as to whether it was a litter or a colony and so forth, but I suppose it could have an impact on reports or shelter views, and might make things messy. I didn't realize I could add a litter reference without creating a litter, so we will do that from here on out. I see I can do that on the bulk change animals and it shows up in the shelter view litter sort. Having the CVS import field will be very helpful when I am doing colonies.

Thank you
